scripts/jupyterkernel/kernelsds/src/main/java/com/kernelsds/app/App.java

```diff
@@ -0,0 +1,42 @@
+package com.kernelsds.app;
+
+
+import io.github.spencerpark.jupyter.channels.JupyterConnection;
+
+import io.github.spencerpark.jupyter.kernel.KernelConnectionProperties;
+import io.github.spencerpark.jupyter.channels.JupyterSocket;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.logging.Level;
+
+public class App 
+{
+    public static void main( String[] args ) throws Exception {
+        if (args.length < 1)
+            throw new IllegalArgumentException("Missing connection file argument");
+
+        Path connectionFile = Paths.get(args[0]);
+
+        if (!Files.isRegularFile(connectionFile))
+            throw new IllegalArgumentException("Connection file '" + connectionFile + "' isn't a file.");
+
+        String contents = new String(Files.readAllBytes(connectionFile));
+
+        //JupyterSocket.JUPYTER_LOGGER.setLevel(Level.WARNING);
+        JupyterSocket.JUPYTER_LOGGER.setLevel(Level.WARNING);
+
+        KernelConnectionProperties connProps = KernelConnectionProperties.parse(contents);
+        JupyterConnection connection = new JupyterConnection(connProps);
```

Comment:
Add detailed comments for readability and debuggability, especially for the code that configure flags.